test/e2e/predictor/test_huggingface_vllm_openvino.py

```diff
@@ -30,7 +30,7 @@
 from ..common.utils import KSERVE_TEST_NAMESPACE, generate
 
 
-@pytest.mark.llm
+@pytest.mark.vllm
```

Comment:
is there any specific reason for this ?

Reply:
to make it different so that pytest can run it separately